Source/OpenTK/Platform/Windows/WglHelper.cs

```diff
@@ -107,24 +107,95 @@ object SyncRoot
 
         IntPtr GetAddress(string function_string)
         {
-            IntPtr address = Wgl.GetProcAddress(function_string);
-            if (!IsValid(address))
+            if (IsGL11(function_string))
             {
-                address = Functions.GetProcAddress(WinFactory.OpenGLHandle, function_string);
+                return Functions.GetProcAddress(WinFactory.OpenGLHandle, function_string);
+            }
+            else
+            {
+                return Wgl.GetProcAddress(function_string);
             }
-            return address;
         }
 
         #endregion
 
         #region Private Members
 
-        static bool IsValid(IntPtr address)
+        static bool IsGL11(string function_string)
         {
-            // See https://www.opengl.org/wiki/Load_OpenGL_Functions
-            long a = address.ToInt64();
-            bool is_valid = (a < -1) || (a > 3);
```

Comment:
Invalid non 1.1 funcptr should only happen when either of these two conditions is met:

- There is no context active (thus no driver can be queried for the export)
- The context does not support the queried function/extension 
  - ex.: extension not supported
  - possibly: query a func that became core with gl3 on a gl2 rather than EXT/ARB/... context  (not sure if drivers are required to be forward compatible here)

But obviously both variants are usage errors of wglGetProcAddress.
The result for an invalid call unfortunately seems undefined.